src/app/functions/server/deleteAccount.ts

```diff
@@ -37,6 +39,21 @@ export async function deleteAccount(
     logger.info("deactivated_onerep_profile", {
       subscriber_id: subscriber.id,
     });
+
+    // try to unsubscribe from subplat
```

Comment:
This function is also called from the `fxa-rp-events` route, in which case we probably don't want to run this. So maybe this needs to be called in the `onDeleteAccount` action separately, or `deleteAccount` could take a parameter to skip it?

Reply:
I think with the updated logic, there's an `if` statement checking for whether subscription is present from fxa side, which is a more fool-proof way of checking if we'd want to skip

Reply:
That's perfect, thanks.